lib/safira/roulette.ex

```diff
@@ -0,0 +1,199 @@
+defmodule Safira.Roulette do
+  @moduledoc """
+  The Roulette context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Safira.Repo
```

Comment:
I know that the warn flag disables the warning regarding Ecto.Query being unused, but why is Ecto.Query being imported in the first place?

Reply:
The roulette.ex file was created alongside the context with `mix phx.gen.context`. This task generates a context with functions around the ecto schemas that compose the context, so none of this was directly created by me. Anyways, since there are no methods that resort to Ecto.Query yet, then that can be removed, even though I think it will need to be imported again later on. 👍